src/include/duckdb/execution/operator/scan/physical_table_scan.hpp

```diff
@@ -61,6 +61,10 @@ class PhysicalTableScan : public PhysicalOperator {
 
 	bool Equals(const PhysicalOperator &other) const override;
 
+	OrderPreservationType OperatorOrder() const override {
```

Comment:
`PhysicalTableScan::SourceOrder()`, `PhysicalTableScan` is not an operator so this has no effect